### PR TITLE
Change chain name to `gmordie`

### PR DESCRIPTION
**Pull request by Tbaut**
self explanatory

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -174,8 +174,8 @@ impl_opaque_keys! {
 
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-    spec_name: create_runtime_str!("template-parachain"),
-    impl_name: create_runtime_str!("template-parachain"),
+    spec_name: create_runtime_str!("gmordie"),
+    impl_name: create_runtime_str!("gmordie"),
```
Comment (arrudagates) [inline runtime/src/lib.rs]:
```suggestion
    spec_name: create_runtime_str!("gm_parachain"),
    impl_name: create_runtime_str!("gm_parachain"),
```

**Review by arrudagates — CHANGES_REQUESTED** [review]
Honestly kept forgetting to change this every time I made a runtime upgrade 😅

File: runtime/src/lib.rs (hunk 1 above)
Comment (arrudagates) [reply runtime/src/lib.rs]: Changing to gm_parachain as GMorDie is the team DAO, which is a separate entity from the parachain itself.

Comment (Tbaut) [reply runtime/src/lib.rs]: aah sounds good :)

**Review by arrudagates — APPROVED** [review]

**Comment by Tbaut** [comment]
Hehe I'm still playing with Signer and Parachain-template definitely didn't look funky enough :D

**Pull request merged**